airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/GenerateInputActivityImpl.java

```diff
@@ -80,6 +83,16 @@ public GeneratedJobInput getSyncWorkflowInput(final SyncInput input) {
 
       final JobRunConfig jobRunConfig = TemporalWorkflowUtils.createJobRunConfig(jobId, attempt);
 
+      LocalDefinitionsProvider provider = new LocalDefinitionsProvider(LocalDefinitionsProvider.DEFAULT_SEED_DEFINITION_RESOURCE_CLASS);
+      List<StandardDestinationDefinition> destinationDefinitionList = provider.getDestinationDefinitions();
```

Comment:
Same concern here about using a definitions provider - we should be reading these things from the database

Reply:
Yep! Thank you. At this point, I changed the value, and in TemporalClient - deleted it. I left the explanation in the comment above.